### ISSUE TITLE
Support of embedded properties?

### ISSUE THREAD

**Issue by carkar** [noamt/elasticsearch-grails-plugin#72]
Having some problems getting embedded properties to work properly. Getting an exception thrown at SearchableClassPropertyMapping.groovy:137. (Property [property] is mapped as component, but dependent mapping is not injected.) when component is set to true. Otherwise it tries to map the property as a String.

Doesn't seem to be any support for this, is there a workaround or is it work in progress?

**Comment by motleycrew** [comment]
Hi @carkar 

I am having exact same issue. Did you find a workaround to the problem?

**Comment by carkar** [comment]
@motleycrew Nope. Moved some properties into domain in combination with using a mysql river instead.
